zenoh/src/net/routing/hat/p2p_peer/interests.rs

```diff
@@ -147,7 +147,8 @@ impl HatInterestTrait for HatCode {
         };
         for dst_face in tables.faces.values_mut().filter(|f| {
             f.whatami == WhatAmI::Router
-                || (options.tokens()
+                || (face.whatami == WhatAmI::Client
```

Comment:
Under which circumstances do routers send peer-to-peer peers interest? A simple grep in `zenoh/src/net/routing/hat` shows only results in `client` and `p2p_peer`.